gammapy/cube/basic_cube.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions to perform basic functions for map and cube analysis.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..maps import WcsNDMap
+
+
+def fill_map_counts(event_list, ndmap):
+    """Fill a ``WcsMap` with events from an EventList.
+
+     The energy of the events is used for a non-spatial axis homogeneous to energy.
+     The other non-spatial axis names should have an entry in the colum names of the ``EventList``
+
+     Parameters
+     ----------
+     event_list : `~gammapy.data.EventList`
+             the input event list
+     ndmap : `~gammapy.maps.Map`
+         Target map
+     """
+    # The list will contain the event table entries to be fed into the WcsNDMap
+    coord_dict = dict()
+    ref_geom = ndmap.geom
```

Comment:
Remove this local variable? It's just used once, and the line there isn't very long yet.